src/pages/index.js

```diff
@@ -5,12 +5,212 @@ import { PopupWithForm } from "../components/PopupWithForm.js";
+
+/****** API CALLS***** */
+
+//CARD SECTION
+
+const opts = {
+	baseUrl: "https://around-api.en.tripleten-services.com/v1",
+	headers: {
+		authorization: "d50bb54b-1efc-4b8a-a5b1-3d5c72d6a1b0",
+		"Content-Type": "application/json",
+	},
+};
+let sectionCards;
+const api = new Api(opts);
+
+/** 1. GET ALL CARDS ROUTE */
+api.getInitialCards()
+	.then((data) => {
+		//handle the response data
+		console.log("Data from Api: ", data); //debugging statement
+
+		// Create a new Section instance with the fetched data and render the items
+		sectionCards = new Section(
+			{
+				items: data,
+				renderer: createCard,
+			},
+			utils.htmlIds.cardList
+		);
+		sectionCards.renderItems();
+	})
+	.catch((err) => {
+		console.log(err); //debugging statement
+		console.error(`Error getting cards: ${err} `);
+	});
+
+api.getUserInfo()
+	.then((res) => {
+		console.log("Inside getUserInfo, Response: ", res); //debugging
+		userInfo.setUserInfo({
+			name: res.name,
+			description: res.about,
+		});
+		userInfo.setProfilePicture({
+			profilePicture: res.avatar,
+		});
+	})
+	.catch((err) => {
+		console.log("ERROR: ", err);
+		console.error(err);
+	});
+
+/**
+ * Param: userData
+ * Description:	Sets the change/edit name/title and the description of the profile
+ * 					UPDATE USER PROFILE INFO ROUTE
+ */
+function handleProfileSubmitForm(userData) {
+	// Call the updateUserInfo method from the Api class
+	api.updateUserInfo(userData.title, userData.description)
+		.then(() => {
+			userInfo.setUserInfo({
+				name: userData.title,
+				description: userData.description,
+			});
+			//userInfo.setProfilePicture( {profilePicture: userData.avatar});
+
+			profileEditForm.close();
+		})
+		.catch((err) => {
+			console.error(`Error updating new information: ${err}`);
+		});
+} //end func
+
+function handleDeleteCard(cardElement, cardId) {
+	/**deleteCardModal open -- need to create HTML element like modal_opened */
+	console.log("inside handle delete card func, cardid: ", cardId); //debugging
+
+	//open delete card modal/popup
+	deleteCardModal.open();
+
+	//submit action
+	deleteCardModal.setSubmitAction(() => {
+		api.deleteCard(cardId)
+			.then(() => {
+				deleteCardModal.close();
+				cardElement.remove();
+			})
+			.catch((err) => {
+				console.log(err);
+				console.error(err);
+			});
+	});
+} //end func
+
+function handleEditProfilePicture(url) {
+	console.log("inside handlePrfilePicture, url: ", url);
+	//open edit profile picture modal (avatar)
+
+	api.updateUserImage({ avatar: url.url })
+		.then((res) => {
+			console.log("inside api.updateUserImage, resposne: ", res); //debugging
+			userInfo.setProfilePicture({ profilePicture: url.url });
+			editProfilePictureForm.close();
+		})
+		.catch((err) => {
+			console.error("Picture Not Update", err);
+			editProfilePictureForm.close();
+		});
+} //end func
+
+function handleCardLike(card) {
+	console.log("CARD: ", card); //debugging
+
+	if (card._isLiked) {
+		api.disLikeCard(card._data._id)
```

Comment:
`_data`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.